From 91086657847112115dbc579d6e3b67f70466b9a2 Mon Sep 17 00:00:00 2001
From: Auto Configured <auto.configured>
Date: Sat, 26 Sep 2015 00:21:08 -0700
Subject: [PATCH] add em560 usb-vid,pid

---
 drivers/usb/serial/option.c |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/usb/serial/option.c b/drivers/usb/serial/option.c
index 9da566a..69321e1 100644
--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -86,6 +86,8 @@ static void option_instat_callback(struct urb *urb);
 #define HUAWEI_PRODUCT_K3765			0x1465
 #define HUAWEI_PRODUCT_K4605			0x14C6
 #define HUAWEI_PRODUCT_E173S6			0x1C07
+/*em560*/
+#define HUAWEI_PRODUCT_EM560   			0x1D09
 
 #define QUANTA_VENDOR_ID			0x0408
 #define QUANTA_PRODUCT_Q101			0xEA02
@@ -578,6 +580,9 @@ static const struct option_blacklist_info telit_le920_blacklist = {
 };
 
 static const struct usb_device_id option_ids[] = {
+	/*em560*/
+	{ USB_DEVICE(HUAWEI_VENDOR_ID, HUAWEI_PRODUCT_EM560) },
+
 	{ USB_DEVICE(OPTION_VENDOR_ID, OPTION_PRODUCT_COLT) },
 	{ USB_DEVICE(OPTION_VENDOR_ID, OPTION_PRODUCT_RICOLA) },
 	{ USB_DEVICE(OPTION_VENDOR_ID, OPTION_PRODUCT_RICOLA_LIGHT) },
-- 
1.7.9.5

